src/datasource.ts

```diff
@@ -315,6 +323,26 @@ export default class CogniteDatasource {
         };
         if (target.tab === Tab.Custom && target.assetQuery.func) {
           query.function = target.assetQuery.func.replace(/ID/g, String(ts.id));
+          const regexSearch = /\[.*?\]/g;
+          const regexMatches = query.function.match(regexSearch);
+          query.aliases = [];
+          for (const match of regexMatches) {
+            // format is id, aggregation, granularity
+            const aliasParts = match
+              .substr(1, match.length - 2)
+              .split(',')
+              .map(x => _.trim(x, ' \'"'));
+            if (aliasParts.length === 1) continue;
+            const alias: DataQueryAlias = {
+              alias: `alias${aliasParts.join('_')}`,
+              id: Number(aliasParts[0]),
+            };
+            alias.aggregate = aliasParts[1];
```

Comment:
This is out of scope for this PR, but it might be nice to make an `Enum` with the aggregates as some point as they are static.